### PR TITLE
Lint fixes

### PR DESCRIPTION
**Pull request by axelstudios**
#### What's this PR do?
Fixes issues detected by eslint

### CODE REVIEW

File: seed/static/seed/js/controllers/delete_cycle_modal_controller.js [hunk 1]
```diff
@@ -73,13 +73,13 @@ angular.module('BE.seed.controller.delete_cycle_modal', [])
...
+            function successHandler () {
               $scope.delete_cycle_status = 'success';
               $scope.uploader.in_progress = false;
             }
-            function errorHandler(err) {
-              console.error('Failed to delete cycle: ');
-              console.error(err);
+            function errorHandler (err) {
+              // console.error('Failed to delete cycle: ');
+              // console.error(err);
```
Comment (perryr16) [inline seed/static/seed/js/controllers/delete_cycle_modal_controller.js:82]: Any particular reason to hide these console errors?

Comment (axelstudios) [reply seed/static/seed/js/controllers/delete_cycle_modal_controller.js:82]: They currently get flagged by eslint - Ideally we'd be using a logging class that doesn't console.log anything in production, and just presents all errors on the page in a toast/modal.  I'm open to alternatives

**Review by perryr16 — APPROVED** [review]
That's a lot of semicolons! Looks good to me. Merge when ready

**Comment by perryr16** [comment]
Will eslint be added to the github PR automated checks?

**Pull request merged**